src/main/views/alphabetical-search.njk

```diff
@@ -0,0 +1,81 @@
+{% from "govuk/components/table/macro.njk" import govukTable as 'govukTable' %}
+
+{% extends "template.njk" %}
+{% block pageTitle %}
+  {{ "Court List" | safe }}
+{% endblock %}
+{% block beforeContent %}
+  {% from "govuk/components/back-link/macro.njk" import govukBackLink %}
```

Comment:
we have the common components macro that we import for back buttons and/or submit buttons on generic pages
use:
```
{{ goBack('/search-option') }}
```

Reply:
Agree with this although 507 will fail without 506 in that case as common components were defined on the 506 branch

Reply:
Merged in 506 - and now updated to use the common components back button